textproc/pandoc/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        jgm pandoc 3.6.2
+github.setup        jgm pandoc 3.7.0.2
 # Change github.tarball_from to 'releases' or 'archive' next update
```

Comment:
please cat upon this comment. It will require you to update the checksums as well.

Reply:
Thanks, done.